baseapp/abci.go

```diff
@@ -552,3 +554,19 @@ func splitPath(requestPath string) (path []string) {
 
 	return path
 }
+
+func (app *BaseApp) ListSnapshots(abci.RequestListSnapshots) abci.ResponseListSnapshots {
```

Comment:
Do we plan on supporting state sync for stargate?

Reply:
yes, there is a separate PR that integrates it but it's blocked on this PR. anyways these are added as no-ops for now